### ISSUE TITLE
Add GeminiEngine

### ISSUE THREAD

**Issue by zhudotexe** [zhudotexe/kani#35]

**Comment by dcssi** [comment]
I agree that its needed, however in the mean time you can use kani with Gemini using openai api
https://youtu.be/vH2iMV2Y3dI?feature=shared
